styles.css

```diff
@@ -76,17 +81,25 @@ form {
   display: flex;
   flex-direction: column;
   position: absolute;
-  top: 15%;
+  top: 10%;
   left: 15%;
-  line-height: 300%;
-  font-size: 18px;
+  right: 15%;
+  bottom: 10%;
+  line-height: 250%;
+  font-size: 125%;
```

Comment:
I'm curious about why you chose to use percentages for font size here and a few other places. It's not wrong or anything! I'm just wondering about your reasoning. Have you learned about rem vs em vs px vs percentages yet? It might be worth a quick Google!

Reply:
I chose percentages so that when I make my window smaller, it would adjust to the window size. But I think it is not working that way. I will Google!

Reply:
You're correct! Font size when given as a percentage doesn't really scale the same as giving a `<div>` a percentage, for example. Check out the docs here for an explanation: https://developer.mozilla.org/en-US/docs/Web/CSS/font-size